### PR TITLE
Standalone sensor groundwork

### PR DESCRIPTION
**Pull request by iximiuz**
Sensor got two new flags (the exact names to be refined though):

- `-mode=controlled|standalone`
- `-command=<JSON-encoded MonitorStart command>`

The default mode is 'controlled' and the sensor's behavior in that mode
(supposedly) stayed unchanged.

In the 'standalone' mode, the sensor becomes independent from the driver app
(docker-slim). The sensor terminates as long as the target app is terminated
and the artifacts have been persisted.

Additionally, the signal forwarding (from the sensor's process to the
target app) is implemented, including sending SIGKILL to the target app
after a grace period. Beware, though, that the signal forwarding works
only in the standalone mode.

This PR is part of a series of (potentially stacked) PRs with a common theme "Standalone Sensor Mode".

### CODE REVIEW

**Comment by kcq** [comment]
my fault for not having consistent channel var names (a mix of *Ch and *Chan; should be one)

File: pkg/app/sensor/app.go [hunk 1]
```diff
@@ -66,17 +66,17 @@ func getCurrentPaths(root string) (map[string]interface{}, error) {
 	return pathMap, nil
 }
 
-func startMonitor(errorCh chan error,
+func startMonitor(
+	errorChan chan error,
 	startAckChan chan bool,
-	stopWork chan bool,
-	stopWorkAck chan bool,
-	pids chan []int,
```
Comment (kcq) [inline pkg/app/sensor/app.go:73]: pids is a placeholder for more targeted monitoring (will revisit this again later)

Comment (iximiuz) [reply pkg/app/sensor/app.go:73]: Should I put the `pids` arg back?

Comment (kcq) [reply pkg/app/sensor/app.go:73]: no need for now

**Pull request merged**